src/routes/hud/elements/targethud/TargetHud.svelte

```diff
@@ -1,9 +1,8 @@
 <script lang="ts">
-    import ArmorStatus from "./ArmorStatus.svelte";
     import {listen} from "../../../../integration/ws.js";
     import type {PlayerData} from "../../../../integration/types";
     import {REST_BASE} from "../../../../integration/host";
-    import {fly} from "svelte/transition";
+    import {fade, fly} from "svelte/transition";
```

Comment:
Unused import fly, replace the transitions please.

Reply:
> Unused import fly, replace the transitions please.

bro trying to be blaubeere

Reply:
alright this one is correct though